config/routes.rb

```diff
@@ -102,7 +102,7 @@
     resources :bundles do
       get :manage, on: :member
       resources :bundle_products, controller: "bundle_products", except: [:show]
-      resources :file_uploads, path: "files", only: [:new, :create, :destroy]
+      resources :file_uploads, path: "files", only: [:index, :create, :destroy]
```

Comment:
For reference, here's where the mistake was introduced: https://github.com/tablexi/nucore-open/commit/2a6483fbbc0eb0423e8d29479fe5e246adcbcf58